src/alfasim_score/converter/alfacase/base_operation.py

```diff
@@ -50,224 +46,172 @@
-                            FLUID_WATER: Array([0.0], FRACTION_UNIT),
-                        },
-                    ),
-                ),
-                velocities=InitialVelocitiesDescription(
-                    position_input_type=TableInputType.length,
-                    table_length=VelocitiesContainerDescription(
-                        positions=Array([0.0], LENGTH_UNIT),
-                        velocities={
-                            FLUID_GAS: Array([0.0], VELOCITY_UNIT),
-                            FLUID_OIL: Array([0.0], VELOCITY_UNIT),
-                            FLUID_WATER: Array([0.0], VELOCITY_UNIT),
-                        },
-                    ),
-                ),
-                temperatures=InitialTemperaturesDescription(
-                    position_input_type=TableInputType.length,
-                    table_length=TemperaturesContainerDescription(
-                        positions=Array([0.0, well_length.GetValue()], LENGTH_UNIT),
-                        temperatures=Array(
-                            [
-                                formation_data["temperatures"].GetValues(TEMPERATURE_UNIT)[0],
-                                operation_data["flow_initial_temperature"].GetValue(
-                                    TEMPERATURE_UNIT
-                                ),
-                            ],
-                            TEMPERATURE_UNIT,
-                        ),
-                    ),
+            NodeDescription(
+                name=GAS_LIFT_MASS_NODE_NAME,
+                node_type=NodeCellType.MassSource,
+                pvt_model=self._get_fluid_model_name(),
+                mass_source_properties=MassSourceNodePropertiesDescription(
+                    temperature_input_type=MultiInputType.Constant,
+                    source_type=MassSourceType.AllVolumetricFlowRates,
+                    volumetric_flow_rates_std={
+                        FLUID_GAS: NULL_VOLUMETRIC_FLOW_RATE,
+                        FLUID_OIL: NULL_VOLUMETRIC_FLOW_RATE,
+                        FLUID_WATER: NULL_VOLUMETRIC_FLOW_RATE,
+                    },
                 ),
             ),
-        )
-
-    def build_output_profiles(self) -> List[ProfileOutputDescription]:
-        """Build the output profiles data for the well."""
-        return [
-            ProfileOutputDescription(
-                curve_names=self.default_output_profiles,
-                location=OutputAttachmentLocation.Main,
-                element_name=WELLBORE_NAME,
-            )
         ]
+
+    def configure_annulus(self, alfacase: CaseDescription) -> None:
+        """
+        Configure the annulus data.
+        Default configuration is done by the alfacase converter.
+        """
+        pass
+
+    def generate_operation_alfacase_description(self) -> CaseDescription:
+        """Generate the configured node with data of the current operation."""
+        alfacase_configured = deepcopy(self.base_alfacase)
```

Comment:
Is it necessary to make a copy?

Reply:
I wanna make sure I'm not persisting any modification in the `base_alfacase` data. Instead of a copy I could just build a new base alfacase i.e. `alfacase_configured = self.alfacase_converter.build_base_alfacase_description()`, in the case I need to call `generate_operation_alfacase_description` twice (e.g in a sequence of operations with same base alfacase), but I think it could cost more time than just do a deep copy.